german/the-black-hack.md

```diff
@@ -251,7 +251,7 @@ Magische Waffen erhöhen üblicherweise den Eigenschaftswert um +1 für jede Pro
 
 Magische Rüstung verdoppelt die Menge an RP, die die Rüstung normalerweise haben würde, etwa hätte ein magisches Kettenhemd 12 **Rüstungspunkte**. Sie funktionieren nach den üblichen Regeln für **Rüstungspunkte**.
 
-Traditionelle OSR-Spiele bilden magische Gegenstände ab, indem sie +1 oder +2 auf Angriffsproben oder Rüstungsklasse hinzufügen. Wenn der SL für hochstufige Charaktere stärkere magische Rüstungen ausgeben will, kann man sich vorsichtig überlegen, die **Rüstungspunkte** eventuell zu verdreifachen.
+Traditionelle OSR-Spiele bilden magische Gegenstände ab, indem sie +1 oder +2 auf Angriffswürfe oder Rüstungsklasse hinzufügen. Wenn der SL für hochstufige Charaktere stärkere magische Rüstungen ausgeben will, kann man sich vorsichtig überlegen, die **Rüstungspunkte** eventuell zu verdreifachen.
```

Comment:
Ich glaube nicht, dass ich da besonders darauf geachtet habe, einen der Begriffe exklusiv zu verwenden und kann damit leben, wenn beides im Text vorkommt. Wenn es da Bedenken in die eine oder andere Richtung gibt, sollte man sich auf einen Begriff einigen und den dann überall verwenden. Aber wie gesagt, von mir aus sehe ich die Notwendigkeit nicht.